tests/plugins/colorbutton/manual/customclass.html

```diff
@@ -0,0 +1,22 @@
+<textarea id="editor" name="editor">Test text.</textarea>
+
+<script>
+	CKEDITOR.addCss( '.text-red { color: #F00; }' );
+	CKEDITOR.addCss( '.text-blue { color: #00F; }' );
+
+	CKEDITOR.replace( 'editor', {
+		colorButton_colors: 'text-red/F00, text-blue/00F',
+		colorButton_foreStyle: {
+			element: 'span',
+			attributes: { 'class': '#(colorName)' },
```

Comment:
@f1ames, do you have any suggestions how we should name this test?

Reply:
Maybe something like `customForeStyleColor` and `customForeStyleColorName` - ofc all lower caps (which is not super readable, but that's the convention we have).

Reply:
We have some discussion with @hub33k about `#(color)` manual test as it will just produce invalid class names and invalid results, so we decided to skip it.